tests/build/test_builder.py

```diff
@@ -112,7 +112,19 @@ def test_pokeapi_schema_valid(schema):
     assert_that(paths, has_item(
         matches_regexp(r'\/v2\/ability\/\{[\w]+\}')))
 
-#### petstore
+
+def test_pokeapi_schema_valid_replay(schema):
+    pokeapi_schema = build_schema_batch(pokeapi_requests, UpdateMode.REPLAY)
+    paths = list(pokeapi_schema.paths.keys())
+    assert 14 == len(paths)
```

Comment:
Maybe for a bit easier debugging use hamcrest?
```suggestion
    assert_that(paths, has_length(14))
```